.taskcluster.yml

```diff
@@ -69,6 +69,19 @@ tasks:
       assume_scope_prefix: assume:repo:github.com/${github_repository_full_name}
     in:
       $let:
+        cot_artifacts:
```

Comment:
Do we need CoT in our PRs? We only need CoT for doing trust-y things, like signing or pushing to Google Play, right?

Reply:
That's correct, we don't need CoT there. Having it, though, doesn't hurt. Level-3 workers still check the signature of the CoT artifacts. If one doesn't come from a trusted level-3 worker, then it will bail out.

That said, CoT on PRs will become mandatory once we build docker images on Taskcluster.